config/initializers/omniauth.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
   if !Rails.env.development? && !Rails.env.test?
+    provider :lti, :oauth_credentials => {:test => 'secret'}
```

Comment:
I think this line is not needed.  https://github.com/emory-libraries/avalon/blob/master/config/settings.yml#L74-L90

Reply:
Removed.